### PR TITLE
Added section on examining offline containers

### LINKED ISSUE #1046: [doc] Add general section for offline OSD/service management and/or troubleshooting section
### Type of report
   - [ ] bug
   - [X] feature request

### Source Document

https://documentation.suse.com/ses/7.1/single-html/ses-admin/#migrating-db-device

### Summary

The section https://documentation.suse.com/ses/7.1/single-html/ses-admin/#migrating-db-device contains quite useful information on how to query an offline OSD deployed by cephadm. It would help a lot to have some general information about service management for offline containers, not only one specific use-case like the db migration, although one might get the gist of it. See [upstream docs](https://docs.ceph.com/en/latest/cephadm/troubleshooting/#calling-miscellaneous-ceph-tools) for more information.
Maybe it should be in both the admin guide and the troubleshooting guide (at least linked to one of them). I could imagine an own section underneath [operational tasks](https://documentation.suse.com/ses/7.1/single-html/ses-admin/#storage-salt-cluster), something like "Offline container management" or so, I'm not sure what the best approach would be. That would allow to have a couple of examples like the already existing lvm migration as well as querying an OSD or a MON:

```
pacific:~ # cephadm unit stop --name osd.1
# alternatively 
pacific:~ # systemctl stop ceph-<FSID>@osd.<OSD_ID>

pacific:~ # cephadm shell --name osd.1
[ceph: root@pacific /]# ceph-objectstore-tool --data-path /var/lib/ceph/osd/ceph-1/ --op list

####
pacific:~ # cephadm unit stop --name mon.pacific
[ceph: root@pacific /]# ceph-monstore-tool /var/lib/ceph/mon/ceph-pacific/ dump-keys

```

### PR DESCRIPTION
**Pull request by tbazant**
closes #1046

### CODE REVIEW

**Comment by ebl-nde** [comment]
I have two comments:

- If an OSD is stopped you might want to add the 'noout' flag before stopping it (or shortly after).
- For completeness sake I would add how to start the stopped unit again after the maintenance is finished:
pacific:~ # cephadm unit start --name  DAEMON_NAME

**Review by ebl-nde — APPROVED** [review]

File: xml/admin_saltcluster.xml [hunk 1]
```diff
@@ -1679,4 +1679,67 @@ devices                   /dev/vdb
 &prompt.smaster;ceph-salt purge
 </screen>
  </sect1>
+ <sect1 xml:id="ceph-offline-container-management">
+  <title>Offline container management</title>
+  <para>
+   You can run specific commands, for example
```
Comment (dariavladykina) [inline xml/admin_saltcluster.xml:1685]:
```suggestion
   You can run specific commands, for example,
```

File: xml/admin_saltcluster.xml [hunk 2]
```diff
@@ -1679,4 +1679,67 @@ devices                   /dev/vdb
...
+   <command>ceph-objectstore-tool</command> and
+   <command>ceph-monstore-tool</command>, inside stopped containers by calling
+   a &cephadm; shell. The following examples illustrate common use cases:
+  </para>
+  <tip>
+   <para>
+    When stopping an OSD daemon, we recommend to set the <literal>noout</literal>
```
Comment (dariavladykina) [inline xml/admin_saltcluster.xml:1692]:
```suggestion
    When stopping an OSD daemon, we recommend setting the <literal>noout</literal>
```

**Review by dariavladykina — COMMENTED** [review]
Hi, a couple of comments here, thanks.

**Pull request merged**